### DISCUSSION TITLE
Any way to define a response from the echo server while testing get requests?

### DISCUSSION THREAD

**Discussion by mbrondogutierrez** [fortio/fortio#473] [Q&A] [answered]
We are trying to test a product with the echo server behind as a back end.. is there a way to define a specific answer to be returned in the echo server without changing from POST to GET request?

**Comment by ldemailly** [comment, marked as answer]
So you can control the echo server with all the options mentioned on
https://github.com/fortio/fortio#server-urls-and-features

including `-echo-server-default-params` part

What answer do you want / can you give an example?

**Reply by mbrondogutierrez** [reply]
That's exactly what I needed, thank you
